Clean up and update readme (#1501)
Ryujinx/Ryujinx: README.md

</p>

<h5 align="center">

>     As of August 2020, Ryujinx goes past menus and in-game on over 1,500 commercial titles. Of those, more than half are considered playable. See the compatiblity list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.

Reviewer: ```suggestion
    As of August 2020, Ryujinx goes past menus and in-game on over 1,500 commercial titles. Of those, more than half are considered playable. See the compatibility list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.
```

Reviewer: typo/misspelling leftover from my own changes originally.

**Step three:**

Build the App using a Command prompt in the project directory. You can quickly access it by holding shift in explorer (in the Ryujinx directory) then right clicking, and typing the following command:  
Run `dotnet build -c Release -r win-x64` inside the Ryujinx project folder to build Ryujinx binaries.

> Every file related to Ryujinx is stored in the `Ryujinx` folder. This folder is located in the user folder, which can be accessed by clicking `Open Ryujinx Folder` under the File menu in the GUI.

Reviewer: ```suggestion
Ryujinx system files are stored in the `Ryujinx` folder. This folder is located in the user folder, which can be accessed by clicking `Open Ryujinx Folder` under the File menu in the GUI.
```

Reviewer: Clarifies that it is Ryujinx system files stored here. Not all Ryujinx files are stored there, as the program folder with the binaries still exists elsewhere.

## Features

 - **Audio**

>    Audio output is entirely supported, audio input (microphone) isn't supported. We use C# wrappers for [OpenAL](https://openal.org/downloads/OpenAL11CoreSDK.zip) (installation needed), and [libsoundio](http://libsound.io/) as the fallback.

Reviewer: ```suggestion
   Audio output is entirely supported, audio input (microphone) isn't supported. We use C# wrappers for [OpenAL](https://openal.org/downloads/oalinst.zip) (installation needed), and [libsoundio](http://libsound.io/) as the fallback.
```

Reviewer: We want them to install the openAL windows binary, not the core SDK. Not sure why the last version had the wrong link too.

## Help

> If you have homebrew or a particular game marked playable or in-game in our compatibility list that doesn't work within the emulator, you can contact us through our [Discord server](https://discord.gg/N2FmfVc). We'll take note of whatever is causing the app/game to not work, put it on the watch list and fix it at a later date.

Reviewer: ```suggestion
If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, you can contact us through our [Discord server](https://discord.gg/N2FmfVc). We'll take note of whatever is causing the app/game to not work, put it on the watch list and fix it at a later date.
```

Reviewer: General rewording for clarification/accuracy.
